src/plugins/nativestorage.ts

```diff
@@ -2,8 +2,8 @@ import { Cordova, Plugin } from './plugin';
 
 
 /**
- * @name Native Storage
- * @description
+ * @name NativeStorage
```

Comment:
This is the name that appears in the docs. We want to keep the spacing in between the words.